src/plugins/util.commerce.ts

```diff
@@ -138,9 +139,6 @@ export const getFromPrice = (item: Node): Node | undefined => {
       const first = variants.get(0);
       let moneyNode = isTruthy(first.path(['onSale'])) ? first.path(['salePriceMoney']) : first.path(['priceMoney']);
       let price = getAmountFromMoneyNode(moneyNode);
-      if (price === undefined) {
```

Comment:
This condition is never true because `getAmountFromMoneyNode` always returns a Node